src/components/playButton/PlayButton.styles.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components'
+import { colors } from '../../theme/colors'
+import { mediaQueries } from '../../theme/mediaQueries'
+import { space } from '../../theme/space'
+
+export const ButtonWrapper = styled.div`
```

Comment:
As this is a `button`, it should render as one in the DOM (for a11y reasons). This should be `styled.button` instead 🙂when you inspect in the browser, you should see it rendered as a `button` type.